project/src/main/java/com/google/sps/servlets/ResourceHandlerServlet.java

```diff
@@ -70,45 +92,43 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
         System.out.println("You submitted: " + location);
         System.out.println("You submitted: " + link);
         System.out.println("You submitted: " + description);
-        System.out.println("You submitted: " + nine);
-
-        // System.out.println("You submitted: " + ageGroup);
 
         // Datastore code
-        // Datastore datastore = DatastoreOptions.getDefaultInstance().getService();
-        // KeyFactory keyFactory = datastore.newKeyFactory().setKind("Resource");
-        // FullEntity taskEntity =
-        //     Entity.newBuilder(keyFactory.newKey())
-        //         .set("organizerName", organizerName)
-        //         .set("organizerEmail", organizerEmail)
-        //         .set("eventName", eventName)
-        //         .set("eventDate", eventDate)
-        //         .set("location", location)
-        //         .set("link", link)
-        //         .set("description", description)
-        //         .set("ethnicity", ethnicity)
-        //         .build();
-        // datastore.put(taskEntity);
-        // response.sendRedirect("/index.html");
+        Datastore datastore = DatastoreOptions.getDefaultInstance().getService();
+        KeyFactory keyFactory = datastore.newKeyFactory().setKind("Resource");
+        FullEntity taskEntity =
+            Entity.newBuilder(keyFactory.newKey())
+                .set("organizerName", organizerName)
+                .set("organizerEmail", organizerEmail)
+                .set("eventName", eventName)
+                .set("eventDate", eventDate)
+                .set("location", location)
+                .set("link", link)
+                .set("description", description)
+                // .set("ageGroup", ageGroup)
```

Comment:
This should work according to the [docs](https://cloud.google.com/java/docs/reference/google-cloud-datastore/latest/com.google.cloud.datastore.BaseEntity.Builder#com_google_cloud_datastore_BaseEntity_Builder_set_java_lang_String_java_util_List___extends_com_google_cloud_datastore_Value_____)

What's going wrong here?

Reply:
Can't add entire arrayList, so fixed it by asking user to only input 1 selection (temporary solution)